modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/internal/scheduler/ScheduledPublishInfoImpl.java

```diff
@@ -3,16 +3,17 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-package com.liferay.change.tracking.web.internal.scheduler;
+package com.liferay.change.tracking.internal.scheduler;
 
 import com.liferay.change.tracking.model.CTCollection;
+import com.liferay.change.tracking.scheduler.ScheduledPublishInfo;
 
 import java.util.Date;
 
 /**
  * @author Preston Crary
  */
-public class ScheduledPublishInfo {
```

Comment:
We don't need a impl for ScheduledPublishInfo since it's just a plain java class. Simply move the original `ScheduledPublishInfo` to the api module.